examples/2d/bounding_2d.rs

```diff
@@ -350,9 +356,7 @@ fn aabb_cast_system(
         **intersects = toi.is_some();
         if let Some(toi) = toi {
             gizmos.rect_2d(
-                aabb_cast.ray.ray.origin
-                    + *aabb_cast.ray.ray.direction * toi
-                    + aabb_cast.aabb.center(),
+                aabb_cast.ray.ray.origin + *aabb_cast.ray.ray.direction * toi,
```

Comment:
`aabb_cast.aabb.center()` is always `Vec2(0.0, 0.0)`, and so adding it here has no effect on the position of the rendered `rect_2d`